### DISCUSSION TITLE
ValueError: The Planet API Key must be provided.

### DISCUSSION THREAD

**Discussion by coblinski** [opengeos/leafmap#67] [Q&A] [answered]
Hello,

I got this Error when I try to open the 'Planet Imagery' or the 'time slider' in the toolbar. 
ValueError: The Planet API Key must be provided.

Any ideia how can I solve this problem?

Thanks

**Comment by giswqs** [comment, marked as answer]
Look at the instructions in this notebook example: https://leafmap.org/notebooks/22_time_slider

**Reply by coblinski** [reply]
Thank you!!

**Reply by shubhramani21** [reply]
hey do you know how you have got the API from planet.com
